app/src/main/java/pl/kossa/akainotes/adapters/NotesRvAdapter.kt

```diff
@@ -21,6 +22,7 @@ class NotesRvAdapter(
         val note = notes[position]
         holder.titleTextView.text = note.title
         holder.descriptionTextView.text = note.description.take(50)
+        holder.itemView.setOnClickListener { listener(note) }
```

Comment:
lepiej listener.invoke(note), ponieważ jakby np. mógłbyć nullable to wtedy można zrobić coś takiego listenner.?invoke(..)